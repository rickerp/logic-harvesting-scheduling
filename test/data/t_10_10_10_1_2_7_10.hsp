10
2
4 8 3 2 8 6 6 7 10 10
3 2 4 6
7 1 3 4 7 8 9 10
5 2 4 7 8 9
4 1 2 3 5
4 4 6 7 9
5 1 5 7 9 10
7 2 3 5 6 8 9 10
4 2 3 7 9
7 2 3 5 6 7 8 10
4 2 6 7 9
6 1 4 6 4 3 2 1 3 3
2 5 6 1 7 4 7 3 3 2
1
