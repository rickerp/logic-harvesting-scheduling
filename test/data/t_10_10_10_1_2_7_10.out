27
3 4 6 8
4 1 3 5 10
1 9
